charts/ocis/templates/storageusers/deployment.yaml

```diff
@@ -92,9 +92,16 @@ spec:
             - name: STORAGE_USERS_S3NG_REGION
               value: {{ .Values.services.storageusers.storageBackend.driverConfig.s3ng.region | quote }}
             - name: STORAGE_USERS_S3NG_ACCESS_KEY
-              value: {{ .Values.services.storageusers.storageBackend.driverConfig.s3ng.accessKey | quote }}
+              valueFrom:
```

Comment:
I checked the jobs.yaml, and there it seems we're actually missing most S3 settings. Can you add the missing ones and the secret references?

Reply:
Fixed, and added it to the second cornjob as well.